check-deps/__have_libcapstone.c

```diff
@@ -8,6 +8,6 @@
 int main()
 {
 	cs_insn insn;
-	printf("size: %lu\n", sizeof(insn));
+	printf("size: %d\n", sizeof(insn));
```

Comment:
`%zu` might be a better fit.  See [man 3 printf](https://linux.die.net/man/3/printf) for "The length modifier".

Reply:
updated!